docs/src/main/paradox/loglevels/log4j2.md

```diff
@@ -0,0 +1,76 @@
+# Log4j2
+
+Dynamic Log Levels for Log4j2 hooks into Akka Management and provides a route where log levels can be read and set over HTTP.
+
+## Project Info
+
+@@project-info{ projectId="loglevels-log4j2" }
+
+Requires @ref:[Akka Management](../akka-management.md) and that the application uses [Log4j2](https://logging.apache.org/log4j/2.x/) as logging backend.
+
+@@dependency[sbt,Gradle,Maven] {
+symbol1=AkkaManagementVersion
+value1=$project.version$
+group=com.lightbend.akka.management
+artifact=akka-management-loglevels-log4j2_$scala.binary.version$
+version=AkkaManagementVersion
+group2=com.lightbend.akka.management
+artifact2=akka-management_$scala.binary.version$
+version2=AkkaManagementVersion
```

Comment:
some indentation here to make it more readable (as I guess we have in other places)

Reply:
Fixed